pkg/api/vlabs/validate.go

```diff
@@ -1474,6 +1473,30 @@ func (a *Properties) validateContainerRuntime() error {
 	return nil
 }
 
+func (a *Properties) isValidVersion(k8sVersion string) (bool, error) {
```

Comment:
you could rename `k8sVersion` to `minVersion` or something like that, otherwise the function name might make it look like it's validating whether or not "k8sVersion" is valid

Reply:
thanks, renamed to `minVersion`